package.json

```diff
@@ -75,6 +75,7 @@
     "applicationinsights": "1.0.8",
     "chart.js": "^2.9.4",
     "chokidar": "3.5.2",
+    "elliptic": "^6.5.4",
```

Comment:
Mistake? You shouldn't be adding stuff to the root package.json unless you're adding something to core

Reply:
I'll revert this file. I was having some issues getting elliptic to work at some point. I also reverted the yarn.lock file change that contained elliptic@^6.5.4